Sources/Tropos/Views/CircularProgressLayer.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+
+@objc(TRCircularProgressLayer) class CircularProgressLayer: CALayer {
+    @objc var progress: CGFloat = CGFloat(0)
```

Comment:
This could be either `: CGFloat = 0` or `= CGFloat(0)` (I usually prefer the former).